appveyor.yml

```diff
@@ -0,0 +1,59 @@
+version: '{build}'
+skip_branch_with_pr: true
+image: Visual Studio 2017
```

Comment:
Why not 2019?

Reply:
Because I got errors like
`error MSB8020: The build tools for llvm (Platform Toolset = 'llvm') cannot be found. To build using the llvm build tools, pl`
in `Visual Studio 2019`:
https://ci.appveyor.com/project/q4a/gpcs4/builds/32598015/job/3r9lowisy7lboa2t#L206
but 2017 works fine.